playbooks/provisioning/openstack/openstack_dns_records.yml

```diff
@@ -1,25 +1,52 @@
 ---
 - name: "Generate list of private A records"
   set_fact:
-    private_records: "{{ private_records | default([]) + [ { 'type': 'A', 'hostname': hostvars[item]['ansible_hostname'], 'ip': hostvars[item]['openstack']['private_v4'] } ] }}"
+    private_records: >
```

Comment:
As far as I can tell, this (and a lot of other changes in this PR) the same code as before, just formatted differently.

These pollute the actual changes and make the review much harder to do thoroughly. Could you please create a separate PR for formatting changes or at least put them in a separate commit?